swift/WorkflowUI/Tests/WorkflowUITests.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
+class WorkflowUITests: XCTestCase {
```

Comment:
We can get rid of the placeholder, as there are real tests in here.

Reply:
Done